app/Resources/views/default/index.html.twig

```diff
@@ -11,7 +11,7 @@
                 <p>
                     <svg id="icon-status" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1671 566q0 40-28 68l-724 724-136 136q-28 28-68 28t-68-28l-136-136-362-362q-28-28-28-68t28-68l136-136q28-28 68-28t68 28l294 295 656-657q28-28 68-28t68 28l136 136q28 28 28 68z" fill="#759E1A"/></svg>
 
-                    Your application is ready to start working on it at:
+                    You're ready to start working on your application at:
```

Comment:
I would split this in 2 sentences, something along the lines of:

"Your application is ready now. Start working on it at:"